MinPlatformPkg/Test/Library/TestPointCheckLibNull/TestPointCheckLibNull.c

```diff
@@ -635,6 +635,26 @@ TestPointSmmEndOfDxeSmrrFunctional (
   return EFI_SUCCESS;
 }
 
+/**
```

Comment:
If you can reuse the existing APIs, it looks like the changes in this file might not be required.

Reply:
Refactored.